addons/html_editor/static/src/core/sanitize_plugin.js

```diff
@@ -28,4 +29,10 @@ export class SanitizePlugin extends Plugin {
             ADD_ATTR: ["contenteditable"],
         });
     }
+
+    restoreSanitizedContentEditable(root) {
+        for (const node of selectElements(root, ".o_not_editable, .o_editable")) {
+            node.contentEditable = node.matches(".o_editable") ? "true" : "false";
```

Comment:
No need for a ternary operator since `matches` already returns a `boolean` you can use it directly.